assets/src/edit-story/app/prepublish/constants.js

```diff
@@ -482,7 +489,7 @@ export const MESSAGES = {
               }}
             >
               {__(
-                '<a>Use full screen videos and images whenever possible to create a more immersive experience. (more info)</a>',
+                '<a>Use full screen videos and images whenever possible to create a more immersive experience (more info)</a>',
```

Comment:
If the whole string is wrapped in a link, move the link outside the `__()`. Doesn't make to translate it in that case. Also means you won't need `TranslateWithMarkup`

Reply:
Ah nice catch!